code/game/gamemodes/events/powerfruit.dm

```diff
@@ -0,0 +1,90 @@
+//the actual powerfruit obj
+#define POWER_PER_FRUIT 500
+#define SPREAD_CHANCE 15
+#define ATTACK_CHANCE 15
+
+/obj/structure/cable/powerfruit
+	name = "powerfruit"
+	desc = "A strange alien fruit somehow creating power. How curious! It looks dangerous, better not touch it"
```

Comment:
"A strange alien fruit that generates electricity passively. Best not to touch it."

Reply:
> generates electricity passively
"passively generates electricity" is more grammatically correct

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA